### DISCUSSION TITLE
Focus style in footer on social links

### DISCUSSION THREAD

**Discussion by RedaHamouche** [community/community#106407] [Accessibility]
### Select Topic Area

Bug

### Body

I noticed that on the homepage of ("https://github.com/" when we are not signed in),  the social media links don't have focus style, therefore i don't really see when i tab on them. I believe it cause an accessibilité issue.

**Comment by PauGon75** [comment]
q

**Comment by cehfisher** [comment]
Hi @RedaHamouche - thank you for providing this feedback! We wanted to follow up on this reported bug. We were unable to replicate this issue in our review. Is this an issue you can replicate today? If could you please let us know which theme you were using when you encountered this issue, that would be helpful. Thank you! (cc @queenofcorgis)
